README.md

```diff
@@ -88,14 +88,14 @@ Once NPM is installed, you should be able to run:
 
 ## Installation
 
-You'll need at least Ruby v1.9.3 (**v2.1.x** preferred)
+You'll need at least Ruby v1.9.3 (**v2.4.x** preferred)
```

Comment:
Here I think we should just say Ruby 2.4 require as there's some syntax in the app that won't work with Ruby 1.9.

Reply:
done!